packages/terra-popup/src/_PopupOverlay.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+import './PopupOverlay.scss';
+
+const propTypes = {
+  /**
+   * The string representation of the index.
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: [],
+};
+
+class PopupOverlay extends React.Component {
+
+  componentWillMount() {
```

Comment:
On windows firefox, the their scrollbar changes the width of the page, it's not an overlay. So when setting document to overflow: hidden, the width changes on the page by the width of the scrollbar. So my component would layout to a specific point, and then the scrollbar would dissapear, leaving the popup at an incorrect position.